src/components/AnonymousChat/BaseChatItem.tsx

```diff
@@ -9,13 +9,9 @@ import ChatItemTargetText from './child/ChatItemTargetText';
 import dimen from '../../utils/dimen';
 import useChatScreenHook from '../../hooks/screen/useChatScreenHook';
 import {ANONYMOUS, ANONYMOUS_USER} from '../../hooks/core/constant';
-import {
-  BaseChatItemComponentProps,
-  BaseChatItemTypeProps
-} from '../../../types/component/AnonymousChat/BaseChatItem.types';
+import {BaseChatItemComponentProps} from '../../../types/component/AnonymousChat/BaseChatItem.types';
 import {ChatStatus} from '../../../types/database/schema/ChannelList.types';
 import {DEFAULT_PROFILE_PIC_PATH} from '../../utils/constants';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `BaseChatItemTypeProps` has been removed. If this type is not used elsewhere in the code, then this change is fine. However, if it's used and not imported, it could lead to a reference error.
```diff
- import {
-   BaseChatItemComponentProps,
-   BaseChatItemTypeProps
- } from '../../../types/component/AnonymousChat/BaseChatItem.types';
+ import {BaseChatItemComponentProps} from '../../../types/component/AnonymousChat/BaseChatItem.types';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->